cmd/tikv-ctl/src/main.rs

```diff
@@ -2660,12 +2439,7 @@ fn run_ldb_command(cmd: &ArgMatches<'_>, cfg: &TiKvConfig) {
     engine_rocks::raw::run_ldb_tool(&args, &opts);
 }
 
-fn run_sst_dump_command(cmd: &ArgMatches<'_>, cfg: &TiKvConfig) {
-    let mut args: Vec<String> = match cmd.values_of("") {
-        Some(v) => v.map(ToOwned::to_owned).collect(),
-        None => Vec::new(),
-    };
-    args.insert(0, "sst_dump".to_owned());
```

Comment:
Why remove this line? Would you please test if `sst_dump` works?

Reply:
Yes it's tested and works.

Unlike clap's `ArgMatches`, the `external_subcommand` args in structopt contains the subcommand itself. See https://docs.rs/structopt/0.3.23/structopt/#external-subcommands and L1852. Thus there is no need to insert a `"sst_dump"`.
https://github.com/tikv/tikv/blob/3e53d3b342aba694b11686afb1f6964ee61f89b1/cmd/tikv-ctl/src/main.rs#L1852-L1856

Reply:
is it possible that the tikv-ctl defines these two commands and therefore proxy it to rocks with the same args?

Reply:
Currently the arg proxy mechanism is done my `external_command` in `clap` as well as `structopt`. Otherwise you cannot get arbitrary args and flags as-is without a `--` seperator. That is, if we define these two sub-commands, we must write a `--` between `sst-dump` and the first flag: `tikv-ctl sst-dump -- --help`. See also https://github.com/clap-rs/clap/issues/484.